app/importers/eccn_data.rb

```diff
@@ -0,0 +1,86 @@
+require 'open-uri'
+require 'csv'
+require 'digest/md5'
+
+class EccnData
+  include ::Importer
+
+  EXPECTED_CSV_HEADERS = %w(description eccn1 eccn2 eccn3 eccn4 eccn5)
+
+  ENDPOINT = "#{Rails.root}/data/eccns/eccn.descs.2015.05.20.csv"
+
+  COLUMN_HASH = {
+    eccn1: :eccn0,
+    eccn2: :eccn1,
+    eccn3: :eccn2,
+    eccn4: :eccn3,
+    eccn5: :eccn4,
+  }
+
+  def initialize(resource = ENDPOINT)
+    @resource = resource
+  end
+
+  def import
+    rows = CSV.parse(open(@resource, 'r:ISO-8859-1').read)
+
+    if rows[0].map(&:downcase) != EXPECTED_CSV_HEADERS
+      fail "'#{rows[0]}' are not the headers we expect"
+    end
+
+    model_class.index(rows[1..-1].map { |row| process_row(row) })
+  end
+
+  private
+
+  def process_row(row)
+    doc = {}
+    doc[:description] = row[0]
+
+    doc[:eccn0] = row[1]
+    doc[:eccn1] = row[2]
+    doc[:eccn2] = row[3]
+    doc[:eccn3] = row[4]
+    doc[:eccn4] = row[5]
+
+    eccns = row[1..5].compact
+    urls = eccns.map { |eccn| eccn_to_url(eccn) }.uniq
+
+    doc[:url0] = urls[0] rescue nil
+    doc[:url1] = urls[1] rescue nil
+    doc[:url2] = urls[2] rescue nil
```

Comment:
What type of error do you expect to suppress with the trailing `rescue`? As long as `urls` is an array, looking up a non-existent index won't fail; `nil` will be returned. I may be missing something, but I don't think these `rescue`s are necessary.